.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,15 +25,15 @@
 
 Bug fix checklist:
 - [ ] My fix includes a new test that breaks as a result of the bug (if possible).
-- [ ] All new and existing tests pass in both python 2 and python 3.
+- [ ] All new and existing tests pass in Python 3.6, 3.7, and 3.8 for ubuntu and mac-os.
```

Comment:
How about just putting a period after all tests pass. We have the CI checks to make sure nothing will be merged if it doesn't pass on a particular version or OS. This check box is really to remind people to run the tests locally. Getting this specific means we'll have to update it more often.

Reply:
That makes sense. I've reduced the item down to "All new and existing tests pass."